pkg/genericapiserver/config.go

```diff
@@ -495,7 +502,7 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) (secure, insec
 		handler = apiserverfilters.WithRequestInfo(handler, NewRequestInfoResolver(c), c.RequestContextMapper)
 		handler = api.WithRequestContext(handler, c.RequestContextMapper)
 		handler = genericfilters.WithTimeoutForNonLongRunningRequests(handler, c.LongRunningFunc)
-		handler = genericfilters.WithMaxInFlightLimit(handler, c.MaxRequestsInFlight, c.LongRunningFunc)
+		handler = genericfilters.WithMaxInFlightLimit(handler, c.MaxRequestsInFlight, c.MaxMutatingRequestsInFlight, c.LongRunningFunc, c.MutatingFunc)
```

Comment:
> request info is not mature enough to use here (e.g. it's not clear to me how to test it - there are no tests for filter that use it)

not sure what you mean... the way the filter was added was just refactored, but requestinfo parsing has been present since 1.0 and is what authorization uses